doc/releases/changelog-dev.md

```diff
@@ -178,6 +178,9 @@
   instead of the controlled version of the diagonal unitary.
   [(#2525)](https://github.com/PennyLaneAI/pennylane/pull/2525)
 
+* Reverted device defined gradients fix [(#2485)](https://github.com/PennyLaneAI/pennylane/pull/2485), since
```

Comment:
```suggestion
* Updated the gradients fix [(#2485)](https://github.com/PennyLaneAI/pennylane/pull/2485) to only apply to the `strawberryfields.gbs` device, since
```